examples/gatekeeper-no-root/README.md

```diff
@@ -0,0 +1,112 @@
+# gatekeeper: No Root
+
+### Overview
+
+This example demonstrates how to declaratively run the [gatekeeper]
```

Comment:
This is a very useful example, we should having a brief description of the scenario here. Something along the lines of:

This example demonstrates how to run gatekeeper function declaratively to enforce the policy `Containers must not run as root` on resources.